ps2.c

```diff
@@ -200,29 +190,30 @@ mouse_send_state()
 	}
 }
 
-
 void
 mouse_button_down(int num)
 {
 	buttons |= 1 << num;
+	mouse_send_state();
```

Comment:
Pull request #321 removed those function calls, for a good reason.  Why did you add them back?

Reply:
That's essentially a merge error since I completely rewrote ps2.c/ps2.h on my fork. Will fix.

Reply:
Should be cleaned up now. Sorry about that. And the emulator should be even faster still as a result. 😃